src/engine/GroupBy.cpp

```diff
@@ -735,21 +742,19 @@ std::optional<GroupBy::ExplicitlySortedData> GroupBy::checkIfExplicitlySorted(
   auto child = _subtree->getRootOperation()->getChildren().at(0);
   auto columnIndex = child->getVariableColumn(groupByVariable);
 
-  return ExplicitlySortedData{columnIndex};
+  return HashMapOptimizationData{columnIndex};
 }
 
 // _____________________________________________________________________________
 template <size_t OUT_WIDTH, size_t numAggregates>
-bool GroupBy::createResultForSortedSubtree(
-    IdTable* result,
-    const ad_utility::HashMap<
-        ValueId, std::array<AverageAggregationData, numAggregates>>& map) {
+bool GroupBy::createResultFromHashMap(IdTable* result,
+                                      const HashMapType<numAggregates>& map) {
   // Sort by groupBy column
   std::vector<ValueId> sortedKeys;
   for (const auto& val : map) {
     sortedKeys.push_back(val.first);
   }
-  std::sort(sortedKeys.begin(), sortedKeys.end());
+  std::ranges::sort(sortedKeys.begin(), sortedKeys.end());
```

Comment:
```suggestion
  std::ranges::sort(sortedKeys);
```
That's what sonarcloud meant:)
The future is now:)